### ISSUE TITLE
does RED editor support Robot framework 5?

### ISSUE THREAD

**Issue by Ihebellouze** [nokia/RED#442]
I'm working with RED robot editor on eclipse with the robot framework 3.2.2v, I wanted to ask if RED supports the latest version of robot (robot framework 5) or not? or what the change to make in order to support it?

**Comment by Zeckie** [comment]
No, RED does not support RF 5. See https://github.com/nokia/RED/issues/413

**Comment by Ihebellouze** [comment]
yes I understand you, but actually I do have RED now and I'm adding some customized plugins to it in order to adapt it to my desired use, this is why I'm asking if I could add a plugin to it so it could then support the RF 5. ?
